decidim-core/db/migrate/20170123140857_add_avatar_to_user_groups.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvatarToUserGroups < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_user_groups, :avatar, :string
```

Comment:
As discussed offline, I am already doing the same as the users. Organizations cannot change its avatar but we are using one by default.